### ISSUE TITLE
508 Accessibility: Button must have a focus outline

### ISSUE THREAD

**Issue by ghost** [asyncapi/asyncapi-react#958]
We are using Sort Stie to evaluate our site for Section 508 Accessibility compliance.  It is flagging a button does not have a focus outline.

![image](https://github.com/asyncapi/asyncapi-react/assets/86489380/424b7dcb-e256-4515-8dd1-968fc7b08816)

This "Server specific information" is the violating component; I didn't even realize this was a button if it weren't for the Sort Site error!  Seems there is no need to have this be a button since there is the Expand All component?
![image](https://github.com/asyncapi/asyncapi-react/assets/86489380/c3ba54be-f209-48ef-ba1c-811533e21eea)

**Comment by Aditya90456** [comment]
i can fix this

**Comment by Orthodox-64** [comment]
can u assign this issue to me @AceTheCreator

**Comment by blazethunderstorm** [comment]
@AceTheCreator is this issue open to work ?

**Comment by AadarshM07** [comment]
@AceTheCreator , I would like to work on this issue. Can you assign this to me? Thank you!
